automation/test-execution/scripts/bash/run-mteb-missing-tests.sh

```
@@ -0,0 +1,142 @@
+            -e "trust_remote_code=$trust_code"); then
+
+        end_time=$(date +%s)
+        duration=$((end_time - start_time))
+        duration_min=$((duration / 60))
+
+        echo ""
+        echo "✅ PASSED: $(basename $model) - $preset (${duration_min}m)"
+        ((passed++))
+    else
+        echo ""
+        echo "❌ FAILED: $(basename $model) - $preset"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)